.github/renovate.json5

```diff
@@ -0,0 +1,41 @@
+{
+  "extends": [
+    "config:base"
+  ],
+  "automerge": false,
+  "assignees": [
+    "pierluigilenoci"
+  ],
+  "schedule": [
+    "before 5am every day"
+  ],
+  "packageRules": [
+    {
+      "description": "Auto-merge patch updates for all dependencies",
+      "updateTypes": ["patch"],
+      "automerge": true
+    },
+    {
+      "description": "Group all minor updates per package",
+      "updateTypes": ["minor"],
+      "groupName": "all-minor-updates"
+    },
+    {
+      "description": "Group all major updates per package",
+      "updateTypes": ["major"],
+      "groupName": "all-major-updates"
+    }
```

Comment:
```suggestion
    {
      "description": "Auto-merge patch updates for all dependencies",
      "updateTypes": ["patch"],
      "groupName": "all-patch-updates"
      "bumpVersion": "patch"
    },
    {
      "description": "Group all minor updates per package",
      "updateTypes": ["minor"],
      "groupName": "all-minor-updates"
      "bumpVersion": "patch"
    },
    {
      "description": "Group all major updates per package",
      "updateTypes": ["major"],
      "groupName": "all-major-updates"
      "bumpVersion": "patch"
    }
```

Reply:
From experience directly activating "automerge" isn't a great idea. Lets first see how the initial PRs look like and then iterate on it before we add the automerge option

Reply:
added the bumpVersion flag to automatically increase the version in the Chart.yaml

Reply:
@tuunit, where is the flag?